suite-native/transactions/src/components/TransactionsList/TransactionList.tsx

```diff
@@ -152,7 +186,12 @@ export const TransactionList = ({
             areTokensIncluded,
         ),
     );
-    const isLoaderVisible = isLoadingTransactions && transactions.length === 0;
+
+    const areAllTxnsFetched = useSelector((state: TransactionsRootState) =>
```

Comment:
Call of this selector can be moved to the `TransactionsListFooter` so it does not have to be passed as a prop there. It is not used anywhere else in this file.

Reply:
10127190bd